crypto/rsa/rsa_ameth.c

```diff
@@ -328,7 +329,8 @@ static int pkey_rsa_print(BIO *bp, const EVP_PKEY *pkey, int off, int priv)
         goto err;
 
     if (priv && x->d) {
-        if (BIO_printf(bp, "Private-Key: (%d bit)\n", mod_len) <= 0)
+        if (BIO_printf(bp, "Private-Key: (%d bit, %d primes)\n",
+                       mod_len, primes <= 0 ? 2 : primes + 2) <= 0)
```

Comment:
I'd say that in such case `primes` variable should be renamed to denote that these are additional/excess primes. Alternatively one can add 2 directly at line 323...